resizeImage.py

```diff
@@ -0,0 +1,25 @@
+import cv2
+import os
+
+def read(filename):
+	image = cv2.imread(filename, 0)
+	return image
+
+def resize(image, scale):
+	width = image.shape[0]
+	height = image.shape[1]
+	image = cv2.resize(image, (int(height*scale), int(width*scale)))
+	return image
+
+files = os.listdir()
+for i in files :
+	if i[-5:] == ".jpeg":
```

Comment:
I feel like I'm reading a programming marathon code snippet! 😆
```suggestion
	if i.endswith('.jpeg'):
```

Jokes aside, how about moving this to `digital_image_processing/manipulation/resize_image.py`?